src/data/Candidates.ts

```diff
@@ -1,24 +1,14 @@
 import {Candidate, CandidateID} from "../types";
+import { Hidalgo } from "./Hidalgo";
+import { LePen } from "./Le_Pen";
+import { Melenchon } from "./Melenchon";
+import { Zemmour } from "./Zemmour";
+import { Pecresse } from "./Pecresse";
```

Comment:
Individual candidates objects are created in different files and added to the `candidates Map` here.
Thus when we have ~40 propositions files stay readable. If we get a PR for a candidate it is also easier to check that nothing else was changed :)